.gitignore

```diff
@@ -32,3 +32,6 @@ out
 
 # Prisma
 prisma/generated/client/
+
+# typesense server data
+typesense-data/
```

Comment:
use your local git config `~/.config/git/ignore` globally or `.git/info/exclude` in repo